apps/site/pages/docs/customizing/components.mdx

```diff
@@ -26,24 +26,36 @@ This is how Prices looks like in our current homepage (with the default color bl
 
 1. In your `custom-theme.scss` theme file , add the `data-fs-price` and  `data-fs-price-variant` data attributes with the `Listing` value:
```

Comment:
@renatamottam  renatinha, can you pls review this [page](https://faststore-site-git-docs-review-updates-docs-faststore.vercel.app/docs/customizing/components)